lib/hanami/environment.rb

```diff
@@ -140,12 +140,16 @@ class Environment
     # When initialized, it sets standard `ENV` variables for Rack and Hanami,
     # such as `RACK_ENV` and `HANAMI_ENV`.
     #
-    # It also evaluates configuration from `.env` and `.env.<environment>`
+    # It evaluates configuration ONLY from `.env.<environment>` file
     # located under the config directory. All the settings in those files will
     # be exported as `ENV` variables.
     #
-    # The format of those `.env` files is compatible with `dotenv` and `foreman`
-    # gems.
+    # Master .env file is ignored to suggest clear separation of environment
+    # configurations and discourage putting sensitive information into source
+    # control.
+    #
+    # The format of those `.env.<environment>` files is compatible with `dotenv`
+    # and `foreman` gems.
```

Comment:
dotenv and foreman, yes, but it's also compatible with systemd EnvironmentFile format and probably a lot of others, I'd add that it's a universal unix environment variable declaration format